src/state/slices/opportunitiesSlice/types.ts

```diff
@@ -52,10 +52,8 @@ export type UserStakingOpportunity = {
     | readonly []
 }
 
-// The AccountId of the staking contract in the form of chainId:accountAddress
-export type StakingId = Nominal<string, 'StakingId'> & AssetId
-// The AccountId of the LP contract in the form of chainId:accountAddress
-export type LpId = Nominal<string, 'LpId'> & AssetId
+export type StakingId = Nominal<string, 'StakingId'>
+export type LpId = Nominal<string, 'LpId'>
```

Comment:
An OpportunityId being an AssetId is not going to necessarily be true anymore.

Since we're not dealing with contracts or yield-bearing assets, but rather with native assets, using the assetId as a StakingId would be hacky - removed in `LpId` as well for consistency

Reply:
Discussed this one out, let's use IDs. Unless there's another native asset staking opportunity we integrate (e.g ETH2 staking), there will never be any collision so let's not over-engineer this and make this diff a monster